features/steps/steps.py

```diff
@@ -221,3 +249,41 @@ def step_impl(context, related, relating, other_entity, condition):
                         errors.append(instance_structure_error(inst, rel.RelatingObject))
 
     handle_errors(context, errors)
+
+@then('The relative placement of that {entity} must be provided by an {other_entity} entity')
+def step_impl(context, entity, other_entity):
+    if getattr(context, 'applicable', True):
+        errors = []
+        for obj in context.instances:
+            if not obj.ObjectPlacement.is_a(other_entity):
```

Comment:
`obj.ObjectPlacement` may be None

Reply:
Added a check for it.

Reply:
In other branches Johan added a do_try function, maybe it'll be worth it to use in cases like these?
```
def do_try(fn, default=None):
    try: return fn()
    except: return default
```
In this particular case, it would be
`for obj in f.by_type("IfcElement"): do_try(lambda: obj.is_a('IfcStair'), False)`

But the check you have now seems to work fine as well :) But found it nice to share/perhaps in the future this can be reused

Reply:
Just noticed that we use do_try in our function to get csv file paths, so why not reuse it here. 
Implemented, thank you for the good tip.